tools/docker-compose.yaml

```diff
@@ -3,13 +3,13 @@ version: '3'
 services:
   consul:
     container_name: consul
-    image: consul:latest
+    image: consul:1.5.3
```

Comment:
Let's keep latest here.

Reply:
I would stick to this version as this is our current target, with latest there is no control what is actually used

Reply:
I'm for repeatable builds as well. So I would pick a version and stick with it.